docs/mast/mast.rst

```diff
@@ -477,7 +546,40 @@ The TESS Input Catalog (TIC, and Disk Detective Catalog can also be queried base
                 J165949.90-054300.7 ... https://talk.diskdetective.org/#/subjects/AWI0005ckk
                 J170314.11-035210.4 ... https://talk.diskdetective.org/#/subjects/AWI0005ckv
 
-                
+
+The PanSTARRS catalog also accepts additional parameters to allow for query refinement. These options include column selection,
+sorting, column criteria, page size and page number. Additional information on PanSTARRS queries may be found
+`here <https://catalogs.mast.stsci.edu/docs/panstarrs.html>`__.
+
+Columns returned from the query may be submitted with the columns parameter as a list of column names.
+
+The query may be sorted  with the sort_by parameter composed of either a single column name (to sort ascending),
+or a list of multiple column names and/or tuples of direction and column name (ASC/DESC, column name).
+
+To filter the query, criteria per column name are accepted. The 'AND' operation is performed between all
+column name criteria, and the 'OR' operation is performed within column name criteria. Per each column name
+parameter, criteria may consist of either a value or a list. The list may consist of a mix of values and
+tuples of criteria decorator (min, gte, gt, max, lte, lt, like, contains) and value,
```

Comment:
this line ends with a comma.  Could you add an example of a list of values + criteria tuples?

Reply:
Updated to end on a period as well as updated the example for the simple use of a criteria decorator.